### ISSUE TITLE
To Bootstrap, or not Bootstrap?

### ISSUE THREAD

**Issue by andrerpena** [gearz-lab/react-ui#3]
In issue #2, you @masbicudo proposed a project structure that contemplates both Bootstrap and not Bootstrap components. I totally agree, given that we have the exact same components with the exact same API for both 

The problem right now is that we don't have enough time. I'd rather have a rich set of components for Bootstrap  (or not) then supporting both and have them half-baked.

Then I propose start off having Bootstrap only components. Here's why:

**Bootstrap con:**

Bootstrap isn't that flexible. However, from my personal experience, I had a bad experience with bootstrap when I wanted to create websites with a visual identity. If I was to create a Facebook like website, I wouldn't go with bootstrap, because it would require a ridiculous ammount of CSS overrides to make it unique. But Bootstrap is very suitable for line-of-business apps because they don't need to be **that** unique and it offers good support for forms.

**Bootstrap pro:**

If we didn't go with Bootstrap, we'd have to implement our own presentation framework. I mean, we'd have to implement common CSS to make sure all the components share a concise look and feel. It's not just about styling each component. I's also about making sure they look consistent. And with Bootstrap we'd have that for free.

So, what do you think about going with Bootstrap for now and then trying to make BS independent component, if we feel the need?

**Comment by masbicudo** [comment]
One other HUGE advantage of bootstrap: it already has got tons of themes.

**Comment by andrerpena** [comment]
Right.. Then we agree to start on Bootstrap and then do whatever feels right.

**Comment by masbicudo** [comment]
We can start with bootstrap, but that means only using their CSS, not the JS part.
All components must be `reactjs`, using our gearz guidelines for components development.

**Comment by andrerpena** [comment]
I'm good with using only CSS..
